lib-src/rvm/rvm/rvm_utils.c

```diff
@@ -249,7 +249,6 @@ static void free_list_entry(list_entry_t *cell)
              }); /* end free_list_lock crit sec */
 }
 
-#ifdef UNUSED
 /* clear free lists */
 void clear_free_list(struct_id_t id /* type of free list */)
```

Comment:
`clear_free_list` is only used in this file, only reference outside of `rvm_utils.c` is the prototype in `rvm_private.h`. There is a confusingly similar named `clear_free_lists` also defined in `rvm_utils.c` which is used in `rvm_init.c`.